modules/sdk-coin-apt/package.json

```diff
@@ -42,8 +42,10 @@
   "dependencies": {
     "@aptos-labs/ts-sdk": "^1.32.0",
     "@bitgo/sdk-core": "^28.18.0",
+    "@bitgo/sdk-lib-mpc": "^10.1.0",
```

Comment:
I think i can be moved to dev dependencies

Reply:
Done